source/common/formatter/substitution_formatter.cc

```diff
@@ -1684,11 +1684,29 @@ HeadersByteSizeFormatter::formatValue(const Http::RequestHeaderMap& request_head
       extractHeadersByteSize(request_headers, response_headers, response_trailers));
 }
 
+GrpcStatusFormatter::Format GrpcStatusFormatter::parseFormat(absl::string_view format) {
+  if (format.empty()) {
+    return GrpcStatusFormatter::CamelString;
+  }
+
+  if (format == "CAMEL_STRING") {
```

Comment:
You can probably collapse both ifs into one:

if (format.empty() || (format == "CAMEL_STRING")) {
    return GrpcStatusFormatter::CamelString;
}

Reply:
Done.